### PR TITLE
Add all missing cypress tests

### PR DESCRIPTION
**Pull request by RainingComputers**

### CODE REVIEW

File: meiki_server/main.go [hunk 1]
```diff
@@ -14,8 +15,16 @@ import (
 	"go.uber.org/zap"
 )
 
+func getEnv(key, defaultValue string) string {
```
Comment (alexsanjoseph) [inline meiki_server/main.go:18]: You had to write a function for this? #justgothings

File: meiki_ui/cypress/support/commands.js [hunk 2]
```diff
@@ -1,23 +1,77 @@
...
+    const creds = { username, password }
+
+    cy.testRequest("DELETE", "/auth/delete", creds, false)
+    cy.clearLocalStorage()
+    cy.testRequest("POST", "/auth/create", creds, true)
+})
+
+Cypress.Commands.add("simulateServerDown", (endpoint) => {
+    cy.intercept(endpoint, (req) => {
+        req.destroy()
```
Comment (alexsanjoseph) [inline meiki_ui/cypress/support/commands.js:75]: brilliant

**Comment by alexsanjoseph** [comment]
Simple and elegant. LGTM

File: meiki_server/main.go (hunk 1 above)
Comment (RainingComputers) [reply meiki_server/main.go:18]: Yep

**Pull request merged**